pkg/version/version.go

```diff
@@ -56,16 +56,26 @@ type MeshInfo []ServerInfo
 // NodeType decides the responsibility of the proxy serves in the mesh
 type NodeType string
 
+const userFacingGateway = "gateway"
+
 func ToUserFacingNodeType(t model.NodeType) NodeType {
 	switch t {
 	case model.Router:
-		return "gateway"
+		return userFacingGateway
 	case model.Ztunnel, model.Waypoint, model.SidecarProxy:
 		return NodeType(t)
 	}
 	return ""
 }
 
+func NodeTypeIsGateway(t NodeType) bool {
```

Comment:
I think we'd better include these functions to be the methods of `NodeType`, which can make the code easier to read